website/docs/integrator-section/webapp-tutorial/revoke-document-store-webapp.md

```diff
@@ -1,31 +1,43 @@
 ---
 id: revoke-document-store-webapp
-title: Revoke Document with Web app
-sidebar_label: Revoke Document Store
+title: Revoking Document with Webapp
+sidebar_label: Revoking Document Store
 ---
 
-This guide will help you to revoke a hash using one of the available web application. You will find more details about revoking on the [main guide page](/docs/integrator-section/verifiable-document/ethereum//revoking-document)
+This guide will help you revoke a hash using one of the available web applications. You will find more details about revoking on [this page](/docs/integrator-section/verifiable-document/ethereum//revoking-document).
```

Comment:
Same issue of breaking the flow as we redirect to the ethereum section.

Reply:
added an "Additional reading" section at the end of the page.